challenge_take_out_the_garbage/src/challenge_take_out_the_garbage/take_out_the_garbage.py

```diff
@@ -0,0 +1,101 @@
+                                                 surface_designator=trashbin_designator,
+                                                 area_description="on_top_of")
+        trashbin_designator2 = ds.EdEntityDesignator(robot=robot,
+                                                     id=CHALLENGE_KNOWLEDGE.trashbin_id2)
+        trash_designator2 = DefaultGrabDesignator(robot=robot,
+                                                  surface_designator=trashbin_designator2,
+                                                  area_description="on_top_of")
+        drop_zone_designator = ds.EdEntityDesignator(robot=robot, id=CHALLENGE_KNOWLEDGE.drop_zone_id)
+
+        arm_designator = self.empty_arm_designator = ds.UnoccupiedArmDesignator(robot, {}, name="empty_arm_designator")
+
+        with self:
+
+            # Start challenge via StartChallengeRobust
+            smach.StateMachine.add("START_CHALLENGE_ROBUST",
+                                   states.StartChallengeRobust(robot, CHALLENGE_KNOWLEDGE.starting_point),
+                                   transitions={"Done": "SAY_START_CHALLENGE",
+                                                "Aborted": "SAY_START_CHALLENGE",
+                                                "Failed": "SAY_START_CHALLENGE"})
+
+            smach.StateMachine.add("SAY_START_CHALLENGE",
+                                   states.Say(robot, "I will start cleaning up the trash", block= True),
+                                   transitions={'spoken': "PICK_UP_TRASH"}) # 'TAKE_OUT'})
+
+            smach.StateMachine.add("PICK_UP_TRASH", PickUpTrash(robot=robot, trashbin_designator=trashbin_designator,
+                                                                arm_designator=arm_designator),
+                                   transitions={"succeeded": "ANNOUNCE_END",
+                                                "failed": "ANNOUNCE_END"})
+
+            smach.StateMachine.add("DROP_DOWN_TRASH",
+                                   DropDownTrash(robot=robot, trash_designator=trash_designator,
+                                                 drop_designator=drop_zone_designator),
+                                   transitions={"succeeded": "ANNOUNCE_END",
+                                                "failed": "failed",
+                                                "aborted": "aborted"})
+
+            smach.StateMachine.add("TAKE_OUT",
+                                   TakeOut(robot=robot, trashbin_designator=trashbin_designator,
+                                           trash_designator=trash_designator, drop_designator=drop_zone_designator,
+                                           arm_designator=arm_designator),
+                                   transitions={"succeeded": "ANNOUNCE_TASK",
+                                                "aborted": "aborted",
+                                                "failed": "ANNOUNCE_END"})
+
+            if trashbin_designator2 is not None:
```

Comment:
This is always True, because you declare it at line 31. In case it isn't declared. This will create an error.